restore-with-rid/test.json

```diff
@@ -7,7 +7,6 @@
   "type": "bash",
   "cleanup": true,
   "ignoredRIDs":[
-    "linux-s390x",
-    "linux-ppc64le"
+    "linux-s390x"
```

Comment:
If it works for ppc64le, it's also going to work with s390x, so let's remove that too.

I don't know why this test was disabled on these architectures.

If the test is still expected to fail on previous versions of .NET, we should add a `skipWhen`.

Reply:
I will check with .net7 and update accordingly

Reply:
Thanks. If it fails, can you share the output?

Reply:
the error that I get with .net7 is as below 
```
/home/dotnet-regular-tests/restore-with-rid/restore-with-rid.csproj : error NU1101: Unable to find package Microsoft.NETCore.App.Runtime.linux-ppc64le. No packages exist with this id in source(s): nuget.org
/home/dotnet-regular-tests/restore-with-rid/restore-with-rid.csproj : error NU1101: Unable to find package Microsoft.AspNetCore.App.Runtime.linux-ppc64le. No packages exist with this id in source(s): nuget.org
  Failed to restore /home/dotnet-regular-tests/restore-with-rid/restore-with-rid.csproj (in 1.21 sec).
Process Exit Code: 1
```

Reply:
Updated condition to use SkipWhen

With .net8
restore-with-rid                                            [PASS]      (9s)
with .net7
restore-with-rid                                            [SKIP]      (1s)

Reply:
ok, the error says: I can't find the runtime package needed to publish a self-contained application.
This no longer fails with .NET 8 because it comes with the SDK in a pack.
Let's add a comment to the skipWhen section so we remember this.